deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
```

Comment:
I *think* task needs to be run before starting the geoprocessing service if we use `setuid mmw`:

https://github.com/WikiWatershed/model-my-watershed/blob/1a836bb5f8281b7d7f2b0bedb2a2f7bf9131794a/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml#L2

Looks like it was brought in here:

https://github.com/WikiWatershed/model-my-watershed/blob/develop/deployment/ansible/roles/model-my-watershed.celery-worker/meta/main.yml#L3

I'll add it to the list of explicit geoprocessing dependencies and then leave the order the same as it was before.